### PR TITLE
Disable sending slack notif to core slack channel and matks

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Disable sending slack notif to core slack channel and matks

@Progi1984 After checking with the PrestaShop SA core-team, we don't feel anymore the need to be pinged everyday by presthubot so I disable this after we agreed to. Thank you for the work, it was great to have it for some time 👍 but today we don't find it relevant anymore

### CODE REVIEW

File: src/App/Command/SlackNotifierCommand.php [hunk 1]
```diff
@@ -149,44 +138,28 @@ protected function configure()
...
-
-        // Check PR to review
-        $slackMessageCore[] = $this->checkPRReadyToReview();
-
-        // Need module releases
-        $slackMessageCore[] = $this->checkModuleReadyToRelease();
-
-        // Need module improvements
-        $slackMessageCore[] = $this->checkModuleImprovements();
-
```
Comment (Progi1984) [inline src/App/Command/SlackNotifierCommand.php:182]: Please remove all unuseful methods/

File: src/App/Service/Slack.php [hunk 2]
```diff
@@ -14,7 +14,6 @@ class Slack
         'eternoendless' => '<@U5GRYPEUC>',
         'jolelievre' => '<@UC4KB9BJS>',
         'kpodemski' => 'kpodemski',
-        'matks' => '<@UB61HUD2A>',
         'matthieu-rolland' => '<@UKW0VAT8S>',
         'NeOMakinG' => '<@UQNF13DAR>',
         'Progi1984' => '<@UL16KUPC5>',
```
Comment (Progi1984) [inline src/App/Service/Slack.php:19]:
```suggestion
```

**Review by Progi1984 — COMMENTED** [review]
Two small comments

File: src/App/Command/SlackNotifierCommand.php (hunk 1 above)
Comment (matks) [reply src/App/Command/SlackNotifierCommand.php:182]: Are you sure? I removed only the notification sending and the message, but the other methods contain valuable logic that could be used later.

Comment (Progi1984) [reply src/App/Command/SlackNotifierCommand.php:182]: If it's not used, it's dead. The logic and its code can be found thanks to Git & Github.

Comment (matks) [reply src/App/Command/SlackNotifierCommand.php:182]: Done in https://github.com/PrestaShop/presthubot/pull/52/commits/1d626007a61718b39af65664468544e9e4b47039

File: src/App/Service/Slack.php (hunk 2 above)
Comment (matks) [reply src/App/Service/Slack.php]: Done in https://github.com/PrestaShop/presthubot/pull/52/commits/f5bea383f4c9e0f4829104fc64ae82f82735ca69

**Comment by matks** [comment]
@Progi1984 All feedbacks applied

**Pull request merged**